### PR TITLE
[FIX]Corpus fix from_numpy and from_list; modify widget to work with corpuses without text_features

### PR DESCRIPTION
**Pull request by PrimozGodec**
##### Issue

Fixes https://github.com/biolab/orange3-text/issues/625

##### Description of changes
- Fix from_numpy which didn't initialize text_features and other variables correctly
- The Corpus widget showed an error when the `text_features` list was empty and didn't initialize Used and unused features fields. Fix to initialize fields anyways. Now error only present when no text features in the domain
- Remove deprecated `addSpace` attribute in widgetBoxs.

##### Includes
- [X] Code changes
- [X] Tests
- [ ] Documentation

### CODE REVIEW

File: orangecontrib/text/widgets/tests/test_owcorpus.py [hunk 1]
```diff
@@ -286,6 +287,40 @@ def test_keep_selected_variables(self):
...
 
+    def test_no_text_feature(self):
+        """
+        Test with data which have empty text_features. Widget should not show
+        the error but, should have all features unused.
+        """
+        # widget already loads book-excerpts from file and store context
+        # settings this call restore context settings to default otherwise
+        # Text variable is moved to used_attributes by the context
+        self.widget.settingsHandler.reset_to_original(self.widget)
```
Comment (PrimozGodec) [inline orangecontrib/text/widgets/tests/test_owcorpus.py:298]: @ajdapretnar do you maybe know is there a simpler way to reset settings?

Comment (ajdapretnar) [reply orangecontrib/text/widgets/tests/test_owcorpus.py:298]: Whoa, no idea. It looks ok to me. 🤷

File: orangecontrib/text/widgets/tests/test_owcorpus.py [hunk 2]
```diff
@@ -11,7 +11,8 @@
 
 class TestOWCorpus(WidgetTest):
     def setUp(self):
-        self.widget = self.create_widget(OWCorpus)
+        print("---")
```
Comment (ajdapretnar) [inline orangecontrib/text/widgets/tests/test_owcorpus.py:14]: Is this print intentional?

**Comment by ajdapretnar** [comment]
This works well, but I'd prefer to resolve the comments in code before merging.

File: orangecontrib/text/widgets/tests/test_owcorpus.py (hunk 2 above)
Comment (PrimozGodec) [reply orangecontrib/text/widgets/tests/test_owcorpus.py]: Removed. :)

**Pull request merged**